CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.3 (2019-11-19)
+
+- Fixed the summary action bar component to properly style the correct modal when two modals are on a page but only one has a summary action bar. [#25](https://github.com/blackbaud/skyux-action-bars/pull/25)
```

Comment:
Instead of `properly style`, should this say something more specific like `applies the proper padding to the footer`?

Reply:
I like that. What do you guys think of:
`Fixed the summary action bar component to apply the proper padding in modal footers when two modals are open and only one has a summary action bar.`?

Reply:
Ok. I made a change. How does it look now?

Reply:
@blackbaud-johnly technically we are removing padding and we only want to do it for one modal footer when this happens. Any way you could glance at the change I made and maybe tweak that to how you like it now?